lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -10,7 +10,6 @@ class Base
           ::Errno::ENOENT,
           ::Racc::ParseError,
           ::RubyParser::SyntaxError,
-          ::RuntimeError,
```

Comment:
Ah you know what, I think you're right. This rescue isn't related.

I think you're right about the ThreadPool issue - I think we may not be raising on errors in child threads. 

Maybe I should bring back this attempt to change that behavior: https://github.com/codeclimate/codeclimate-duplication/pull/117